storage/jellyfish_merkle/src/node_type/mod.rs

```diff
@@ -271,26 +271,26 @@ impl From<LeafNode> for Node {
 
 impl Arbitrary for InternalNode {
     type Parameters = ();
+    type Strategy = BoxedStrategy<Self>;
+
     fn arbitrary_with(_args: ()) -> Self::Strategy {
         hash_map(any::<Nibble>(), any::<Child>(), 1..=16)
             .prop_filter(
-                "InternalNode with only one child which is a leaf is illegal",
+                "InternalNode with only one leaf child is illegal",
```

Comment:
o this sounds not right.. >_< will rephrase again.

Reply:
"child leaf" is better

Reply:
"InternalNode constructor panics when its only child is a leaf."

how about this